src/pages/Checkout/CheckoutPage.vue

```diff
@@ -506,7 +514,7 @@ export default {
 			// Doing nothing here allows the normal link handling to happen, which will send the user to /ui-login
 		},
 		guestCheckout() {
-			console.log('Guest checkout method triggered.');
+			this.checkingOutAsGuest = true;
```

Comment:
should this be a togglable method? meaning can they switch away from guestCheckout after selecting it?

Reply:
There isn't a button or link to back out of the guest checkout process in any part of the flow I can see, but refreshing the page would reset them.